src/cocotb_tools/makefiles/simulators/Makefile.vcs

```diff
@@ -58,9 +58,15 @@ ifdef VERILOG_INCLUDE_DIRS
 endif
 
 ifeq ($(PYTHON_ARCH),64bit)
-    EXTRA_ARGS += -full64
+    COMPILE_ARGS += -full64
 endif
 
+# link in libraries even if the linker thinks the library is not needed
+COMPILE_ARGS += -LDFLAGS -Wl,--no-as-needed
```

Comment:
Why is that needed, i.e. what error message did you get otherwise?

Reply:
```
/usr/bin/ld: .../linux64/lib/libvcsnew.so: undefined reference to `vfs_set_dir_map'
/usr/bin/ld: .../linux64/lib/libvcsnew.so: undefined reference to `snpsReallocFunc'
/usr/bin/ld: .../linux64/lib/libvcsnew.so: undefined reference to `snps_mem_get_info'
....
```

Reply:
Can you add the error message to the comment in the Makefile, so we remember why we did it next time we look at it?